src/components/PageTitle/index.module.css

```diff
@@ -6,6 +6,10 @@
   color: #045cfc;
 }
 
+.regular-font {
```

Comment:
May I suggest `.with-tittle` instead? It explains the difference between this and the default in `.root`.

Reply:
Sure

Reply:
Will change it on the component props also

Reply:
It's `tittle`, with 2 `t`s, it's the name of the dot in the i. :pray:

Reply:
🙃 my bad